testcases/kernel/syscalls/recvmmsg/Makefile

```diff
@@ -0,0 +1,10 @@
+#	Copyright (c) Google LLC, 2018
+#	SPDX-License-Identifier: GPL-2.0-or-later
+
+top_srcdir ?= ../../../..
+
+include $(top_srcdir)/include/mk/testcases.mk
+
+LDLIBS += -lpthread
+
```

Comment:
The only supported way how to link with pthreads on Linux is to add CFLAGS += -pthread